src/mp4box/co64.rs

```diff
@@ -49,6 +49,11 @@ impl<R: Read + Seek> ReadBox<&mut R> for Co64Box {
         let (version, flags) = read_box_header_ext(reader)?;
 
         let entry_count = reader.read_u32::<BigEndian>()?;
+        if u64::from(entry_count) > size.saturating_sub(4) / 8 {
```

Comment:
One initial thought which applies to most of this PR: typically, I try to either write code where magic numbers are documented implicitly using a constant or variable name or explicitly in a comment. An example for this line:

```
let box_header_size = mem::size_of::<u32>();
let entry_size = mem::size_of::<u64>();
if u64::from(entry_count) > size.saturating_sub(box_header_size) / entry_size {
```

Now, I realize that the author of this library hasn't exactly taken this approach elsewhere.. so I'm not sure what they prefer. OTOH, the PR is a bit difficult to review as-is, since I have to make these calculations myself in order to verify them. I'll leave it up to you, considering the above, but what do you think?

Reply:
Another idea to make these types of comparisons more readable:

```Rust
let box_header_size = mem::size_of::<u32>();
let entry_size = mem::size_of::<u64>();
let possible_entries_size = size.saturating_sub(box_header_size);
if u64::from(entry_count) > possible_entries_size / entry_size {
```

Reply:
fixed in [df4e2a7](https://github.com/privacyresearchgroup/mp4-rust/pull/2/commits/df4e2a72af19b7388691830f68834fa63be848b5)